### PR TITLE
Add css-loader@2.2.0 w/ npm auto-update

### LINKED ISSUE #10222: [Request] Add css-loader
Library name: https://github.com/raphaelfabeni/css-loader
Git repository url: https://github.com/raphaelfabeni/css-loader
**npm package url(optional):** 
License(s): MIT
Official homepage: http://www.raphaelfabeni.com.br/css-loader/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #10222 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/raphaelfabeni/css-loader
 * Official website (optional, not the repository): http://www.raphaelfabeni.com.br/css-loader/
 * NPM package url (optional): https://www.npmjs.com/package/pure-css-loader
 * License and its reference: MIT, ref: https://github.com/raphaelfabeni/css-loader#license
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 34
   - Count of stars: 809
   - Count of forks: 86
 * NPM download stats (optional):
   - Downloads in the last day: 20
   - Downloads in the last week: 177
   - Downloads in the last month: 1064

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #10222, cc @raphaelfabeni

### CODE REVIEW

**Comment by qwe19272375** [comment]
@pvnr0082t Please help me review this PR, thanks!
I'm not sure if every files in `dist/` have to be collected in the library or not.

File: ajax/libs/css-loader/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "type": "git",
+    "url": "git://github.com/raphaelfabeni/css-loader.git"
+  },
+  "keywords": [
+    ""
+  ],
+  "description": "Simple loaders for your web applications using only one div and pure CSS ♥.",
+  "author": {
+    "name": "Raphael Fabeni"
+  },
```
Comment (pvnr0082t) [inline ajax/libs/css-loader/package.json]: We can use `string` instead of object.

File: ajax/libs/css-loader/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "name": "css-loader",
+  "filename": "css-loader.css",
+  "version": "2.2.0",
+  "homepage": "http://www.raphaelfabeni.com.br/css-loader/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/raphaelfabeni/css-loader.git"
+  },
+  "keywords": [
+    ""
```
Comment (pvnr0082t) [inline ajax/libs/css-loader/package.json]: @qwe19272375 Please enrich the keywords.

**Comment by qwe19272375** [comment]
@pvnr0082t keywords added.
Please help me review this PR, thanks@

**Pull request merged**